spec/controllers/application_controller/timezone_spec.rb

```diff
@@ -33,7 +33,7 @@
 
       context "without a timezone" do
         it "with a system default" do
-          user = FactoryGirl.create(:user)
+          user = FactoryGirl.create(:user, :settings => {})
```

Comment:
@kbrock 
In this test the settings is empty by default, the other tests in the spec explicitly set the settings.